dll/win32/iphlpapi/iphlpapi_main.c

```diff
@@ -2620,12 +2621,113 @@ SetIpForwardEntryToStack(PMIB_IPFORWARDROW pRoute)
 }
 
 DWORD WINAPI
-NhGetInterfaceNameFromDeviceGuid(DWORD dwUnknown1,
-                                 DWORD dwUnknown2,
-                                 DWORD dwUnknown3,
+NhGetInterfaceNameFromDeviceGuid(GUID * pInterfaceGUID,
+                                 PWCHAR pInterfaceName,
```

Comment:
Some annotations would make this more straightforward. E.g. `_Out_` here helps the reader understand at a glance. Even better, `_Out_writes_bytes_to_(*pOutBufLen, *pOutBufLen)`, would exactly specify the relationship with pOutputBufLen.

Reply:
Agree, changed.